src/main/java/org/opensearch/knn/jni/FaissService.java

```diff
@@ -233,6 +233,20 @@ public static native void createByteIndexFromTemplate(
      */
     public static native long loadIndexWithStream(IndexInputWithBuffer readStream);
 
+    /**
+     * Load an index into memory via a wrapping having Lucene's IndexInput with ADC
+     *
+     * @param readStream IndexInput wrapper having a Lucene's IndexInput reference.
+     * @return pointer to location in memory the index resides in
+     */
+    public static native long loadIndexWithStreamADC(IndexInputWithBuffer readStream);
```

Comment:
Do we need both of these functions?

Reply:
No, only `loadIndexWithStreamADCParams` for the `FaissService`. Will remove `loadIndexWithStreamADC`.